File: src/JavaScript/easy/math/problem2520.js
/**
 * @param {number} num
 * @return {number}
 */

var countDigits = function (num) {
    let count = 0
    num.toString().split("").forEach(n => {
        if(num % Number(n) == 0){
            count++
        }
    })
    return count
};

// let num = 7
// let num = 121;
let num = 1248;

countDigits(num);

File: src/Python/easy/math/problem2520.py


class Solution:
    def countDigits(self, num: int) -> int:
        count = 0
        for n in str(num):
            if num % int(n) == 0:
                count += 1
        return count


        
num = 1248

solution = Solution()
solution.countDigits(num)